tests/unit/test_mips.py

```diff
@@ -0,0 +1,94 @@
+import mips_api
+
+import os
+
+import boto3
+import pytest
+from botocore.stub import Stubber
+
+
```

Comment:
how about a test to make sure logout always happens?

Reply:
good call